src/plugins/home/server/ui_settings.ts

```diff
@@ -28,3 +29,14 @@ export const uiSettings: Record<string, UiSettingsParams> = {
     requiresPageReload: true,
   },
 };
+
+export const searchOverviewPageUISetting: Record<string, UiSettingsParams> = {
+  [SEARCH_WORKSPACE_DISMISS_GET_STARTED]: {
```

Comment:
Instead of ui settings, shall we simply store it in local storage? I feel local storage makes more sense to me for a UI behavior

Reply:
we want to persist the settings and even if when they switch browser and devices.